frontend/src/pages/Landing/Home/LandingPage.tsx

```diff
@@ -496,13 +496,6 @@ export const LandingPage = (): JSX.Element => {
                   <OrderedListIcon index={4} />
                   Collect responses at your email address
                 </ListItem>
-                <ListItem textStyle="body-2">
```

Comment:
There are also unused imports for `Link` and `OGP_FORMSG_COLLATE` in this file. perhaps we should remove them as well

Reply:
e9bcfd4e2